airbyte-integrations/connectors/source-public-apis/source_public_apis/public_apis.yaml

```diff
@@ -0,0 +1,70 @@
+version: "0.1.0"
+
+definitions:
+  schema_loader:
+    type: JsonSchema
+    file_path: "./source_public_apis/schemas/{{ options['name'] }}.json"
+  selector:
+    type: RecordSelector
+    extractor:
+      type: DpathExtractor
+      field_pointer: []
+  requester:
+    type: HttpRequester
+    name: "{{ options['name'] }}"
+    http_method: "GET"
+  retriever:
+    type: SimpleRetriever
+    $options:
+      url_base: https://api.publicapis.org/
+    name: "{{ options['name'] }}"
+    primary_key: "{{ options['primary_key'] }}"
+    record_selector:
+      $ref: "*ref(definitions.selector)"
+    paginator:
+      type: NoPagination
+  base_stream:
+    type: DeclarativeStream
+    schema_loader:
+      $ref: "*ref(definitions.schema_loader)"
+    retriever:
+      $ref: "*ref(definitions.retriever)"
+      record_selector:
+        extractor:
+          field_pointer: []
+      requester:
+        $ref: "*ref(definitions.requester)"
+      paginator:
+        type: NoPagination
+
+streams:
+  - $ref: "*ref(definitions.base_stream)"
+    $options:
+      name: "services"
+      field_pointer: ["entries"]
+      path: "/entries"
+      primary_key: "API"
+    retriever:
+      $ref: "*ref(definitions.base_stream.retriever)"
+  #ISSUE: Categories transformations are affected by https://github.com/airbytehq/airbyte/issues/17957
```

Comment:
This is affected by https://github.com/airbytehq/airbyte/issues/17957

Reply:
The issue is closed now, can you try again?

Reply:
All good now I went with Python CDK instead